tf2/README.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+[tf2](http://wiki.ros.org/tf2) is ROS's newest library for keeping track of robot transforms between coordinate frames. Documentation for tf2 with Python is very limited, so this README is intended to document the use of tf2 transforms for Python 2.7 and provide some helpful resources.

```

Comment:
I'm not really understanding what this is contributing - whats wrong with the python tutorials located at http://wiki.ros.org/tf2/Tutorials